### PR TITLE
refactor BitmapUtils

### PR DESCRIPTION
**Pull request by theScrabi**
refactor BitmapUtils

### CODE REVIEW

File: src/com/owncloud/android/utils/BitmapUtils.java [hunk 1]
```diff
@@ -114,75 +117,68 @@ private static int calculateSampleFactor(Options options, int reqWidth, int reqH
...
+
+            switch(orientation) {
+                case ExifInterface.ORIENTATION_FLIP_HORIZONTAL:
+                    matrix.postScale(-1.0f, 1.0f);
+                    break;
+                case ExifInterface.ORIENTATION_ROTATE_180:
+                    matrix.postRotate(180);
+                    break;
+                case ExifInterface.ORIENTATION_FLIP_VERTICAL:
+                    matrix.postRotate(-90);
```
Comment (davigonz) [inline src/com/owncloud/android/utils/BitmapUtils.java]: This line is not included in the previous changes for this case

File: src/com/owncloud/android/utils/BitmapUtils.java [hunk 2]
```diff
@@ -114,75 +117,68 @@ private static int calculateSampleFactor(Options options, int reqWidth, int reqH
...
-            // 8
-            else if (orientation == ExifInterface.ORIENTATION_ROTATE_270)
-            {
-                matrix.postRotate(270);
-            } 
-            
+
             // Rotate the bitmap
-            resultBitmap = Bitmap.createBitmap(bitmap, 0, 0, bitmap.getWidth(), bitmap.getHeight(), matrix, true);
+            final Bitmap resultBitmap = Bitmap.createBitmap(bitmap, 0, 0, bitmap.getWidth(), bitmap.getHeight(), matrix, true);
```
Comment (davigonz) [inline src/com/owncloud/android/utils/BitmapUtils.java]: This line is too long, split it into two different ones

**Comment by davigonz** [comment]
@theScrabi a few minor changes...

File: src/com/owncloud/android/utils/BitmapUtils.java (hunk 1 above)
Comment (theScrabi) [reply src/com/owncloud/android/utils/BitmapUtils.java]: xD copy pasta error

**Comment by davigonz** [comment]
Code approved, ready to test

**Comment by davigonz** [comment]
@jesmrec regarding what you could test here, this utils class is being used to decode and perform other operations over the user avatar and thumbnails shown in the side menu and the list of files (images)

**Pull request merged**